lib/translator/verify.cc

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "verify.h"
+namespace translator {
+
+StatusCode VerifyToNvme(absl::Span<const uint8_t> scsi_cmd,
+                        nvme::GenericQueueEntryCmd& nvme_cmd) {
+  scsi::Verify10Command verify_cmd{};
+  if (!ReadValue(scsi_cmd, verify_cmd)) {
+    DebugLog("Malformed Verify Command");
```

Comment:
I would elaborate more on these debug logs so it is clear why the command is malformed. This way, it is easier to figure out what went wrong. In particular since you are using the same debug log further down.